.github/workflows/unittests.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.9', '3.10']
+        python-version: ['3.9', '3.10', '3.11']
```

Comment:
there are other files (like `python-linting.yml`, `python-formatting.yml`, ...) with 3.9 and 3.10. Should we also update them?

And there are some files (like `shellcheck.yaml` and `conftest.yml`) that just have one python version. Should we update them to be 3.11 (because is faster)? If so, I would change the jobs their from  matrix to explicit, like this https://github.com/mateusoliveira43/python-project-template/blob/main/.github/workflows/cd.yml

Reply:
fixed, thanks